girderformindlogger/api/v1/user.py

```diff
@@ -378,7 +378,7 @@ def updateIDCode(self, id, code):
     @access.public(scope=TokenScope.USER_INFO_READ)
     @autoDescribeRoute(
         Description('Remove an ID Code from a user.')
-        .param('id', 'Profile ID', required=True)
+        .param('id', 'Profile ID', required=True, paramType='path')
```

Comment:
@henryrossiter @shnizzedy We can remove "required=True" option, because it is path param and it is required by default. But it's unnecessary though. Everything else looks good!